data/json/construction.json

```diff
@@ -7109,6 +7109,133 @@
     "pre_terrain": "t_sand_underground",
     "post_special": "done_extract_maybe_revert_to_dirt"
   },
+  {
+    "type": "construction",
+    "id": "constr_clear_rubble",
+    "group": "clear_rubble",
+    "category": "OTHER",
+    "qualities": [ [ { "id": "DIG", "level": 1 } ] ],
```

Comment:
Tools with level 1 of `DIG` quality include digging sticks, trowels, and hoes. I think it would be quite difficult to clear rubble with tools like these, though not impossible. I suppose you meant more real-shovel-like tools in requirements? If yes, then maybe increase them to level 2 of `DIG` quality?

Reply:
i did have it set to `DIG` 2 from the start, i cant remember the reason why i tuned it down

Reply:
it was probably because a lot of this could be cleared by hand and i wanted a way for more `dig` quality to increase speed but compromised with just setting it to `dig` 1 to give access to the most tools available

Reply:
![image](https://user-images.githubusercontent.com/115079701/224540856-6c0ef4fa-c8e7-4fe5-81bc-91349ffc2be8.png)
lvl 2 is for clear rubble so i can change it back to lvl 2 and it shouldn't change anything